rest_framework/request.py

```diff
@@ -14,19 +14,19 @@
 
 from django.conf import settings
 from django.http import HttpRequest, QueryDict
-from django.http.multipartparser import parse_header
 from django.http.request import RawPostDataException
 from django.utils.datastructures import MultiValueDict
 
 from rest_framework import HTTP_HEADER_ENCODING, exceptions
 from rest_framework.settings import api_settings
+from rest_framework.compat import parse_header_params
 
 
 def is_form_media_type(media_type):
     """
     Return True if the media type is a valid form media type.
     """
-    base_media_type, params = parse_header(media_type.encode(HTTP_HEADER_ENCODING))
+    base_media_type, params = parse_header_params(media_type, encoding=HTTP_HEADER_ENCODING)
```

Comment:
Here `params` are ignored, again nobody cares about returned data types in the dict
```suggestion
    base_media_type, _params = parse_header_params(media_type.encode(HTTP_HEADER_ENCODING))
```